localstack/services/awslambda/lambda_executors.py

```diff
@@ -901,28 +948,57 @@ def do_execute():
         invocation_result = InvocationResult(result, log_output=log_output)
         return invocation_result
 
+    def inject_thundra_java_agent(self, func_details, given_opts):
+        if not THUNDRA_JAVA_AGENT_JAR:
+            return
+
+        thundra_apikey = func_details.envvars.get(THUNDRA_APIKEY_ENV_VAR_NAME)
```

Comment:
Let's better extract this into a small util function, to avoid duplication with lines 371-378.